### DISCUSSION TITLE
Privacy Screen for idle users

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3046] [Feature Requests]
I had an idea that if a user is completely idle (eg: no keyboard or mouse activity) for X period of time (eg: 5m), then a privacy screen would be displayed above the current page. Any action (mouse movement or keypress) would instantly disable this. The purpose being that if you leave your workstation for a moment, any sensitive info would be hidden from prying eyes. It's easily defeated, but a nice soft-security measure.

I think this is as easy as a globally z-indexed element with CSS blur and some centered text. Mockup:

![Privacy Screen](https://user-images.githubusercontent.com/522079/94743494-77a71f00-0345-11eb-814a-41e85bba6e79.png)

This could also display the time until you are automatically logged out... if that is a known duration.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
